pkg/objcache/objcache.go

```diff
@@ -27,18 +27,20 @@ import (
 	"time"
 )
 
-// NoExpiry represents caches to be permanent and can only be deleted.
-var NoExpiry = time.Duration(0)
+const (
+	// NoExpiry represents caches to be permanent and can only be deleted.
+	NoExpiry = time.Duration(0)
 
-// DefaultExpiry represents default time duration value when individual entries will be expired.
-var DefaultExpiry = time.Duration(72 * time.Hour) // 72hrs.
+	// DefaultExpiry represents three days time duration when individual entries will be expired.
+	DefaultExpiry = time.Duration(3 * 24 * time.Hour)
 
-// DefaultBufferRatio represents default ratio used to calculate the
-// individual cache entry buffer size.
-var DefaultBufferRatio = uint64(10)
+	// defaultBufferRatio represents default ratio used to calculate the
+	// individual cache entry buffer size.
+	defaultBufferRatio = uint64(10)
 
-// DefaultGCPercent represents default garbage collection target percentage.
-var DefaultGCPercent = 20
+	// defaultGCPercent represents default garbage collection target percentage.
+	defaultGCPercent = 20
```

Comment:
In go this is how you would have done it 

```
package main

func init() {
    objCache.DefaultBufferRatio = 5
    objCache.DefaultGCPercent = 10
}

func main() {
      cache1 := New()
}
```

You can bring the functionality back by adding two additional methods which allow this to be changed at runtime. It is useful for a cache because one needs to really make sure that GC kicks in quickly before running out of memory.